nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
Aha! It's actually quite interesting: it removes options when given filesystem type and device again:
```
root@abbradarbook> mount -t tmpfs tmpfs -o rw,nosuid,nodev,mode=755 testtest                                      /tmp
root@abbradarbook> mount | grep testtest                                                                          /tmp
tmpfs on /tmp/testtest type tmpfs (rw,nosuid,nodev,relatime,mode=755)
root@abbradarbook> mount -o remount,size=3046460k testtest                                                        /tmp
root@abbradarbook> mount | grep testtest                                                                          /tmp
tmpfs on /tmp/testtest type tmpfs (rw,nosuid,nodev,relatime,size=3046460k,mode=755)
root@abbradarbook> mount -t tmpfs tmpfs -o remount,size=3046460k testtest                                         /tmp
root@abbradarbook> mount | grep testtest                                                                          /tmp
tmpfs on /tmp/testtest type tmpfs (rw,relatime,size=3046460k,mode=755)
```